src/trace/context.ts

```diff
@@ -223,9 +223,28 @@ export function readTraceFromLambdaContext(context: any): TraceContext | undefin
     return;
   }
 
-  const traceData = context.clientContext?.custom?._datadog;
+  const custom = context.clientContext?.custom;
 
-  if (!traceData || typeof traceData !== "object") {
+  if (!custom || typeof custom !== "object") {
```

Comment:
maybe a test to validate this condition could be useful ?

Reply:
👍